cloudinit/cmd/main.py

```diff
@@ -799,6 +802,12 @@ def main_modules(action_name, args):
             extra_message="Use `cloud-init init` instead.",
         )
 
+    if name == "final":
+        sys.stderr.write(
```

Comment:
Why should the table should be re-printed if it contains the same information as before?